main/src/modules/systemVisual/iol2opc/src/module.cpp

```diff
@@ -808,8 +816,10 @@ void IOL2OPCBridge::updateOPC()
         if (!unknownObjectInScene)
             onlyKnownObjects.heartBeat();
 
+        objLocOut.write(bStreamObjLoc);
```

Comment:
@Tobias-Fischer @maxime-petit 

This is "bad", because this call to `write` will basically wait until the receiver will get the message.

I warmly suggest to use `BufferedPort` instead.